src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
```

Comment:
Consider adding validation in the `setFruit` method to ensure that the fruit name is not null or empty. This will prevent invalid `FruitTransaction` objects from being created.